demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -676,10 +676,13 @@ class TestAppendOrReplaceCommandInDocs:
     command = 'dxl-send-event'
     old_doc = open(positive_test_data_file).read()
     new_docs = "\n<NEW DOCS>\n"
+    new_command = '\n### dxl-send-event-new-one\n***\nSends the specified event to the DXL fabric.\n##### Base Command\n`dxl-send-event-new-one`'
```

Comment:
this change is unclear, can you explain?

Reply:
Added a new command in the .md file of the test